Tests/test_ProtParam.py

```diff
@@ -14,59 +15,62 @@
 
 
 class ProtParamTest(unittest.TestCase):
+    """Tests for ProtParam."""
+
     def setUp(self):
+        """Initialise objects."""
         self.seq_text = "MAEGEITTFTALTEKFNLPPGNYKKPKLLYCSNGGHFLRILPDGTVDGTRDRSDQHIQLQLSAESVGEVYIKSTETGQYLAMDTSGLLYGSQTPSEECLFLERLEENHYNTYTSKKHAEKNWFVGLKKNGSCKRGPRTHYGQKAILFLPLPV"
         self.analysis = ProtParam.ProteinAnalysis(self.seq_text)
 
     def test_count_amino_acids(self):
-        "Test getting amino acid counts"
+        """Calculating amino acid counts."""
         count_dict = self.analysis.count_amino_acids()
         for i in sorted(count_dict):
             self.assertEqual(count_dict[i], self.seq_text.count(i))
 
     def test_get_amino_acids_percent(self):
-        "Test getting amino acid percentage"
+        """Calculating amino acid percentages."""
         percent_dict = self.analysis.get_amino_acids_percent()
         seq_len = len(self.seq_text)
         for i in sorted(percent_dict):
             self.assertAlmostEqual(percent_dict[i], self.seq_text.count(i) / float(seq_len))
 
     def test_get_molecular_weight(self):
-        "Test calculating protein molecular weight"
+        """Calculating protein molecular weight."""
         self.assertAlmostEqual(round(self.analysis.molecular_weight(), 2),
                                17103.16)
 
     def test_get_monoisotopic_molecular_weight(self):
-        "Test calculating the monoisotopic molecular weight"
+        """Calculating monoisotopic molecular weight."""
         self.analysis = ProtParam.ProteinAnalysis(self.seq_text, monoisotopic=True)
         self.assertAlmostEqual(round(self.analysis.molecular_weight(), 2),
                                17092.61)
 
     def test_get_molecular_weight_identical(self):
-        "Test calculating the protein molecular weight agrees with calculation from Bio.SeqUtils"
+        """Calculating the protein molecular weight agrees with calculation from Bio.SeqUtils."""
```

Comment:
Not related to this PR, but I wonder if this and the following test make sense, since ```molecular_weight``` of ```ProtParam.ProteinAnalysis``` is just a call to the module's function:

```python
 def molecular_weight(self):
        """Calculate MW from Protein sequence."""
return molecular_weight(self.sequence, monoisotopic=self.monoisotopic)
```

Reply:
Not sure, haven't looked at that for a long time.